kbase-extension/static/kbase/js/common/jupyter.js

```diff
@@ -43,10 +43,7 @@ define(['jquery', 'base/js/namespace', 'base/js/dialog'], ($, Jupyter, dialog) =
     }
 
     function getCell(kbaseId) {
-        let cells = getCells(),
-            cell;
-        for (let i = 0; i < cells.length; i += 1) {
-            cell = cells[i];
+        for (const cell of getCells()) {
```

Comment:
Also modernized this, as the linter didn't like it.
I'll take another pass at this module and add tests for it, too, before finishing up. It's small, shouldn't be difficult.